tests/net45/FileTransferTest.cs

```diff
@@ -0,0 +1,128 @@
+﻿using Avalara.AvaTax.RestClient;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tests.Avalara.AvaTax.RestClient.netstandard
+{
+    [TestFixture]
+    public class FileTransferTest
+    {
+        public AvaTaxClient Client { get; set; }
+        public string CompanyCode { get; set; }
+        public CompanyModel TestCompany { get; set; }
+
+        #region Setup / TearDown
+        /// <summary>
+        /// Create a company for use with these tests
+        /// </summary>
+        [SetUp]
+        public void Setup()
+        {
+            try
+            {
+                // Create a client and set up authentication
+                Client = new AvaTaxClient(typeof(TransactionTests).Assembly.FullName,
+                    typeof(TransactionTests).Assembly.GetName().Version.ToString(),
+                    Environment.MachineName,
+                    AvaTaxEnvironment.Sandbox)
+                    .WithSecurity(Environment.GetEnvironmentVariable("SANDBOX_USERNAME"), Environment.GetEnvironmentVariable("SANDBOX_PASSWORD"));
+
+                // Verify that we can ping successfully
+                var pingResult = Client.Ping();
+
+                // Assert that ping succeeded
+                Assert.NotNull(pingResult, "Should be able to call Ping");
+                Assert.True(pingResult.authenticated, "Environment variables should provide correct authentication");
+
+                // Create a basic company with nexus in the state of Washington
+                TestCompany = Client.CompanyInitialize(new CompanyInitializationModel()
```

Comment:
Rather than create a company, can we assert that the account already has a company?